pymc/func_utils.py

```diff
@@ -50,24 +50,26 @@ def find_constrained_prior(
         Lower bound to get `mass` % of probability of `pm_dist`.
     upper : float
         Upper bound to get `mass` % of probability of `pm_dist`.
-    init_guess: Dict[str, float]
+    init_guess : str or float
         Initial guess for ``scipy.optimize.least_squares`` to find the
         optimal parameters of `pm_dist` fitting the interval constraint.
         Must be a dictionary with the name of the PyMC distribution's
         parameter as keys and the initial guess as values.
-    mass: float, default to 0.95
+    mass : float, default to 0.95
         Share of the probability mass we want between ``lower`` and ``upper``.
         Defaults to 95%.
-    fixed_params: Dict[str, float], Optional, default None
+    fixed_params : str or float, optional, default None
         Only used when `pm_dist` has at least three parameters.
         Dictionary of fixed parameters, so that there are only 2 to optimize.
-        For instance, for a StudenT, you fix nu to a constant and get the optimized
+        For instance, for a StudentT, you fix nu to a constant and get the optimized
         mu and sigma.
 
     Returns
     -------
-    The optimized distribution parameters as a dictionary with the parameters'
-    name as key and the optimized value as value.
+    opt_params: dict
```

Comment:
Adding the type is necessary and needed to be fixed. Giving a name to the returned value is optional, but if it is done, it must follow the same conventions as the parameters section

```suggestion
    opt_params : dict
```

Ref: https://pymc-data-umbrella.xyz/en/latest/sprint/tutorials/docstring_tutorial.html#returns-and-yields